biz.aQute.tester/.gitignore

```diff
@@ -1,3 +1,4 @@
 /bin/
 /bin_test/
 /generated/
+/testdir/
```

Comment:
This is bad. If the test need to write something out, it needs to be in `generated/tmp` preferably a unique folder per test method to avoid interactions and allow tests to be concurrent. Writing to a folder outside of generated will mess up the gradle dependency checking since it will see changes in testdir as an input change to the project which will make the project rebuild.

Reply:
This is where the output from `JUnitXMLReporter`/`JUnitXMLListener` goes by default when it executes (as it does when the tests in `biz.aQute.tester` and `biz.aQute.junit.platform` are run). Should be easy to fix.

Reply:
Fixed.